documentation/modules/auxiliary/scanner/mssql/mssql_login.md

```diff
@@ -15,6 +15,176 @@ A docker container can be spun up with the following command to test this module
+    sessions           Quickly switch to another session
+
+
+MSSQL Client Commands
+=====================
+
+    Command            Description
+    -------            -----------
+    query              Run a single SQL query
+    query_interactive  Enter an interactive prompt for running multiple SQL queri
+                       es
+
+
+Local File System Commands
+==========================
+
+    Command            Description
+    -------            -----------
+    getlwd             Print local working directory (alias for lpwd)
+    lcat               Read the contents of a local file to the screen
+    lcd                Change local working directory
+    ldir               List local files (alias for lls)
+    lls                List local files
+    lmkdir             Create new directory on local machine
+    lpwd               Print local working directory
+
+This session also works with the following modules:
+
+  auxiliary/admin/mssql/mssql_enum
+  auxiliary/admin/mssql/mssql_escalate_dbowner
+  auxiliary/admin/mssql/mssql_escalate_execute_as
+  auxiliary/admin/mssql/mssql_exec
+  auxiliary/admin/mssql/mssql_findandsampledata
+  auxiliary/admin/mssql/mssql_idf
+  auxiliary/admin/mssql/mssql_sql
+  auxiliary/admin/mssql/mssql_sql_file
+  auxiliary/scanner/mssql/mssql_hashdump
+  auxiliary/scanner/mssql/mssql_schemadump
+  exploit/windows/mssql/mssql_payload
+```
+
+To interact directly with the session as if in a SQL prompt, you can use the `query` command.
+
+```msf6
+msf6 auxiliary(scanner/mssql/mssql_login) > sessions -i -1
+[*] Starting interaction with 2...
+
+mssql @ 192.168.2.242:1433 (master) > query -h
+Usage: query
+
+Run a single SQL query on the target.
+
+OPTIONS:
+
+    -h, --help      Help menu.
+    -i, --interact  Enter an interactive prompt for running multiple SQL queries
+
+Examples:
+
+    query select @@version;
+    query select user_name();
+    query select name from master.dbo.sysdatabases;
+
+mssql @ 192.168.2.242:1433 (master) > query 'select @@version;'
+Response
+========
+
+    #  NULL
+    -  ----
+    0  Microsoft SQL Server 2022 (RTM) - 16.0.1000.6 (X64)
+	Oct  8 2022 05:58:25
+	Copyright (C) 2022 Microsoft Corporation
+	Developer Edition (64-bit) on Windows Server 2022 Standard 10.0 <X64> (B
+       uild 20348: ) (Hypervisor)
+```
+
+Alternatively you can enter a SQL prompt via the `query_interactive` command which supports multiline commands:
+
+```msf6
+mssql @ 192.168.2.242:1433 (master) > query_interactive -h
+Usage: query_interactive
+
+Go into an interactive SQL shell where SQL queries can be executed.
+To exit, type 'exit', 'quit', 'end' or 'stop'.
+
+mssql @ 192.168.2.242:1433 (master) > query_interactive
+[*] Starting interactive SQL shell for mssql @ 192.168.2.242:1433 (master)
+[*] SQL commands ending with ; will be executed on the remote server. Use the exit command to exit.
+
+SQL >> select @@version;
+
+[*] Executing query: select @@version;
+Response
+========
+
+    #  NULL
```

Comment:
is this NULL expected?

Reply:
Actually yes, but I replaced the query to better showcase multiline functionality